modules/pinentry

```diff
@@ -21,12 +21,12 @@ pinentry_configure := ./configure \
 	--disable-libsecret \
 	--disable-fallback-curses \
 	--disable-pinentry-curses \
-	--disable-pinentry-qt \
```

Comment:
why is this being removed? Doing so seems to introduce a dependency on Qdialog which we're not providing